src/Form/Type/Procuration/AbstractProcurationType.php

```diff
@@ -1,7 +1,10 @@
 <?php
 
-namespace AppBundle\Form;
+namespace AppBundle\Form\Type\Procuration;
```

Comment:
why Procuration is in the `Type`name space and not `GenderType` `UnitedNationsCountryType` ?

Reply:
Because I don't want to move old types in this PR.

Reply:
So don't move the new one now ?

Reply:
This won't be merged in master before I'll make the change everywhere ;).

Reply:
don't keep the target branch in an instable state, its not matter of `master` or `whateverBranch`

Reply:
It’s not instable, it has related form types in a consistent namespace.

Reply:
And broken tests :p

Reply:
please revert the `type` namespace since the other aren't moved

Reply:
Done in https://github.com/EnMarche/en-marche.fr/pull/1948/commits/91858dcc18f0f7249d443173e9d2df9ca794ffb6.